### ISSUE TITLE
Typo in publish from VS

### ISSUE THREAD

**Issue by hypatia1729** [Azure/azure-functions-host#4598]
Hi,

When publishing a .NET Core Functions V2 project from Visual Studio, the publish logs include the entry "`Overwritting extensions.json file with one from build`", in which the word overwriting is misspelt.

Cheers,
Sonja

**Comment by brettsam** [comment]
Hey @soninaren -- do you know where this string would live?

**Comment by fabiocav** [comment]
I think this has been changed.

@hypatia1729 what version of VS, Azure Function extension and Functions SDK are you using?

**Comment by hypatia1729** [comment]
Hi @fabiocav, VS 2019 16.2.0 Preview 4 Enterprise, Functions v2.0. Functions SDK v1.0.29.

**Comment by fabiocav** [comment]
Have you customized your build process by chance? That string is not in any of the products, but it may be found in blog posts and some issues comments from others here with a step to explicitly copy the extensions.json file.

Is this something you have done to your project?

Examples of where this may be found:
https://thatblokecalledjay.com/blog/view/codingdojo/azure-functions-and-dependency-injection-abb17d74076d
https://github.com/Azure/azure-functions-host/issues/3386#issuecomment-454620471
https://github.com/Azure/azure-functions-vs-build-sdk/issues/312#issuecomment-492248255

And a few others. 

If you haven't, is there a chance you can look at the msbuild logs to see where this is coming from? (or provide logs we can inspect *after removing any sensitive information*)

thanks!

**Comment by hypatia1729** [comment]
Aha! You are completely correct.

I had followed the instructions in the first link when playing around with DI in Functions. I've since switched over to the DI using Microsoft.Extensions.DependencyInjection v2.2.0 via Microsoft.Azure.Functions.Extensions (can't upgrade to v3.0 preview 6 as it breaks the Functions extensions). I had left the Directory.Build.targets as per Boris Wilhelms' instructions though.

Thanks for chasing up.
